release-notes/release-notes/5.8-release-notes.md

```diff
@@ -1,39 +1,69 @@
 # 5.8 Release Notes
 
-{% hint style="success" %}
-Cinchy version 5.8 was released on XXXX
-{% endhint %}
+{% hint style="success" %} Cinchy version 5.8 was released on XXXX {% endhint %}
 
 {% hint style="info" %}
 
-For instructions on how to upgrade your platform to the latest version, please review the documentation [here.](../../upgrade-guide/upgrade-guides/)
+For instructions on how to upgrade your platform to the latest version, please
+review the documentation [here.](../../upgrade-guide/upgrade-guides/)
 
 {% endhint %}
+
 ## New Capabilities
 
+### New event-based Source - SOAP API (CDC Triggered)
+
+Cinchy now supports a new Cinchy event-triggered source: SOAP API. This new
```

Comment:
CIN-06700. Still needs documentation in the data-syncs\supported-data-sync-sources\mongodb-collection-cinchy-event-triggered.md